recipes/nanopb/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, replace_in_file
+from os import path
+
+class NanoPbConan(ConanFile):
+    name = "nanopb"
+    version = "0.4.8"
+    license = "zlib"
+    url = "https://jpa.kapsi.fi/nanopb/"
+    description = "Protocol Buffers with small code size"
+    settings = "os", "compiler", "build_type", "arch"
+    exports = '*'
+    generators = "CMakeToolchain", "CMakeDeps"
+    options = {
+        "Shared": [True, False], "fPIC": [True, False],
+    }
+    default_options = {
+        "Shared": False, "fPIC": True,
+    }
```

Comment:
```suggestion
    options = {
        "shared": [True, False],
        "fPIC": [True, False],
    }
    default_options = {
        "shared": False,
        "fPIC": True,
    }
```
the convention is to use lowercase options

Reply:
Only lower case is support by the client :)